app/views/check_csv_file.scala.html

```diff
@@ -29,9 +29,7 @@
 
 @ers_main(title = Messages("ers.upload_csv.page_title"), scripts) {
 
-	<div id="file-wrapper">
 		<p class="font-xsmall"><a href="@routes.CheckingServiceController.checkFileTypePage()" class="link-back">@Html(Messages("ers.back"))</a></p>
-		@header(Messages("ers_check_csv_file.title"))
 		<div id="errors" class="validation-summary visibility" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
 			<h1 class="heading-medium error-summary-heading" id="error-summary-heading">
```

Comment:
This should really be a h2, though if possible the error summary should be the one from the helpers/ui library.